src/control/server/ctl_storage_rpc.go

```diff
@@ -70,8 +69,39 @@ var (
 
 type scanBdevsFn func(storage.BdevScanRequest) (*storage.BdevScanResponse, error)
 
+func ctrlrToPciStr(nc *ctlpb.NvmeController) (string, error) {
+	pciAddr, err := hardware.NewPCIAddress(nc.GetPciAddr())
+	if err != nil {
+		return "", errors.Errorf("Invalid PCI address: %s", err)
```

Comment:
Out of curiosity, why `errors.Errorf()` instead of `errors.Wrap()`?

Reply:
fixed